eng/Version.Details.xml

```diff
@@ -293,6 +293,14 @@
       <Uri>https://github.com/dotnet/corefx</Uri>
       <Sha>0abec4390b30fdda97dc496594f9b1f9c9b20e17</Sha>
     </Dependency>
+    <Dependency Name="System.ComponentModel" Version="4.3.0">
+      <Uri>https://github.com/dotnet/corefx</Uri>
+      <Sha>000000</Sha>
```

Comment:
How will these be corrected?

Reply:
According to https://github.com/aspnet/AspNetCore/blob/master/docs/ReferenceResolution.md#example-adding-a-new-dependency, the `Sha` values don't matter and eventually get updated by the system automatically. `000000` is the suggested value to use when adding a new dependency.

Reply:
This entry can be removed since we will not be flowing dependency updates automatically. This package is obsolete for .NET Core 3 and no longer builds new versions.

Reply:
Thanks for the tip. Have removed this entry.